{"name": "加购物车", "status": "broken", "statusDetails": {"message": "TypeError: 'dict' object is not callable", "trace": "self = <testcases.usercenter.test_user.TestUser object at 0x10abfd580>\nusername = 17380900323, password = 123456\n\n    @allure.story(\"购物车相关\")\n    @allure.title(\"加购物车\")\n    @pytest.mark.parametrize(\"username,password\", get_data['user_login'])\n    def test_shopping_cart(self, username, password):\n        # 登录接口\n        result = login(username, password)\n        token = result.body['token']\n>       param = get_data()['shopping_cart']\nE       TypeError: 'dict' object is not callable\n\ntest_user.py:49: TypeError"}, "attachments": [{"name": "log", "source": "605cf186-8a3e-4bb6-bc60-5081061b863f-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "01355639-5d86-41bd-8364-96737b4c6335-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "17380900323"}, {"name": "password", "value": "123456"}], "start": 1659107399246, "stop": 1659107399423, "uuid": "11a71dab-a2e0-41fb-a526-c0caaf6f22d2", "historyId": "078982f37d6fc574e848a2dd911995ae", "testCaseId": "6732b5a4ef5cebf11d6a36876a07be7c", "fullName": "testcases.usercenter.test_user.TestUser#test_shopping_cart", "labels": [{"name": "feature", "value": "用户中心模块"}, {"name": "story", "value": "购物车相关"}, {"name": "parentSuite", "value": "testcases.usercenter"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestUser"}, {"name": "host", "value": "shallowdeMacBook-Air.local"}, {"name": "thread", "value": "56275-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.usercenter.test_user"}]}